healthcheck/healthcheck_test.go

```diff
@@ -536,6 +536,7 @@ func TestCheckDryrun(t *testing.T) {
 	config := NewConfig(1, &fakeChecker{succeed: false})
 	config.Interval = 250 * time.Millisecond
 	hc.Update(config)
+	time.Sleep(100 * time.Millisecond)
```

Comment:
I don't see this is what you need. It's more reasonable to remove the default case below with a timeout?

Reply:
Makes perfect sense. Just updated.